libraries/chain/webassembly/privileged.cpp

```diff
@@ -145,6 +145,31 @@ namespace eosio { namespace chain { namespace webassembly {
       return context.control.set_proposed_producers( std::move(producers) );
    }
 
+   uint32_t interface::get_wasm_parameters_packed( span<char> packed_parameters ) const {
```

Comment:
I think we should also take a `max_version` and return the `uint32_t` version as the first item in the returned serialization to be consistent with the new format we introduced with `get_kv_parameters_packed`.